test/system/045-start.bats

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bats   -*- bats -*-
+
+load helpers
+
+function setup() {
+    basic_setup
+    run_podman run -d $IMAGE /bin/true
+    cid_none="$output"
+    run_podman run -d --restart=always $IMAGE /bin/true
+    cid_always="$output"
+}
+
+function teardown() {
+    basic_teardown
+}
+
+@test "podman start --all - start all containers" {
+    run_podman start --all
+    is "$output" "$cid_none
+$cid_always"
```

Comment:
This is currently failing; I can't diagnose that. But were it to pass: how certain are you about the order of CIDs? Is podman guaranteed to emit them in consistent order? If not, it gets trickier to test; ask if you need help.

Also, please take advantage of the three-argument form of `is`:
```
   is "$output" "$cid_none...." "podman start --all emits both CIDs"
```